template/prompts.md

```
@@ -1 +1,19 @@
-
+#Role
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)